src/test_data/function_parameter_count_changed.output.ron

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "name": String("function_with_a_parameter_added"),
+        "parameterscount": Uint64(1),
+        "path": List([
+            String("semver_tests"),
+            String("test_cases"),
+            String("parameter_count_changed"),
+            String("function_with_a_parameter_added")
+        ]),
+        "span_begin_line": Uint64(2),
+        "span_filename": String("src/test_cases/parameter_count_changed.rs"),
+        "visibility_limit": String("public")
+    },
+    {
+        "name": String("function_with_parameters_removed"),
+        "parameterscount": Uint64(2),
+        "path": List([
+            String("semver_tests"),
+            String("test_cases"),
+            String("parameter_count_changed"),
+            String("function_with_parameters_removed")
+        ]),
+        "span_begin_line": Uint64(8),
+        "span_filename": String("src/test_cases/parameter_count_changed.rs"),
+        "visibility_limit": String("public")
+    }
+]
```

Comment:
```suggestion
]

```
It's nice to end files in a newline, since that makes it more likely that there won't be any interoperation issues with other programs (more info here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline ).

It's very likely that your editor can be configured to do this automatically when you save the file.